lms/djangoapps/grades/transformer.py

```diff
@@ -119,8 +123,10 @@ def _collect_max_score(cls, block_structure, module):
         Collect the `max_score` from the given module, storing it as a
         `transformer_block_field` associated with the `GradesTransformer`.
         """
-        score = module.max_score()
-        block_structure.set_transformer_block_field(module.location, cls, 'max_score', score)
+        max_score = module.max_score()
+        block_structure.set_transformer_block_field(module.location, cls, 'max_score', max_score)
+        if max_score is None:
+            log.warning("GradesTransformer: max_score is None for {}".format(module.location))
```

Comment:
This will only happen (at most) once per block that gets collected by the grades transformer, right?